modules/oxide/src/less/theme/components/dialog/dialog.less

```diff
@@ -40,11 +40,13 @@
 @dialog-body-text-transform: none;
 
 @dialog-body-link-color: @color-tint;
-@dialog-body-link-text-decoration: none;
-@dialog-body-link-hover-color: darken(@color-tint, 10%);
-@dialog-body-link-hover-text-decoration: none;
-@dialog-body-link-active-color: darken(@color-tint, 10%);
-@dialog-body-link-active-text-decoration: none;
+@dialog-body-link-text-decoration: underline;
+@dialog-body-link-hover-color: darken(@color-tint, 20%);
+@dialog-body-link-hover-text-decoration: underline;
+@dialog-body-link-active-color: darken(@color-tint, 30%);
```

Comment:
Same as with hovers, probably shouldn’t blanket make it all darker. Honestly, I think we’re good with having hover and active the same color since unlike the filled buttons, it is harder to notice that change on a thin lines of a font.